Makefile

```diff
@@ -114,7 +114,4 @@ lint:
 
 .PHONY: lint-fix
 lint-fix:
-	python3 -c "import black" > /dev/null 2>&1 || python3 -m pip install black
-	python3 -c "import isort" > /dev/null 2>&1 || python3 -m pip install isort
-	python3 -m black --target-version py37 .
-	python3 -m isort .
+	ruff --fix .
```

Comment:
Why remove Black?

Please also make sure Ruff is installed.

Reply:
Oops, oversight there.

Reply:
That said, should this actually be `pre-commit run --all-files black ruff`?